### PR TITLE
vet_clinic-DB: Add queries for multiple tables

### PR DESCRIPTION
**Pull request by yodit93**
### In this project, I

- [ ] created owners and species tables,
- [ ] modified animals table to include foreign keys referencing owners and species tables,
- [ ] inserted data to owners and species tables,
- [ ] modified animals table species_id and owners_id columns with their respective values from species and owners table respectively,
- [ ] wrote queries (using JOIN) to answer the following questions:

-         What animals belong to Melody Pond?

-         List of all animals that are pokemon (their type is Pokemon).

-         List all owners and their animals, remember to include those that don't own any animal.

-         How many animals are there per species?

-         List all Digimon owned by Jennifer Orwell.

-         List all animals owned by Dean Winchester that haven't tried to escape.

-         Who owns the most animals?

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -67,3 +67,74 @@ SELECT neutered, count(*) AS escape_count FROM animals GROUP BY neutered;
...
 SELECT species, AVG(escape_attempts) FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 1990 AND 2000 GROUP BY species;
+
+--Modify animals table
+
+--Remove the species column from the animals table.
+ALTER TABLE animals DROP COLUMN species;
+--Add column species_id which is a foreign key referencing species table
+ALTER TABLE animals ADD COLUMN species_id INT REFERENCES species(id);
+--Add column owner_id which is a foreign key referencing owners table
+ALTER TABLE animals ADD COLUMN owner_id INT REFERENCES owners(id);
```
Comment (Munsa1) [inline queries.sql]: - [ ] Kindly put these statements into the `schema.sql` as they are altering the structure of the database.

File: queries.sql [hunk 2]
```diff
@@ -67,3 +67,74 @@ SELECT neutered, count(*) AS escape_count FROM animals GROUP BY neutered;
...
+UPDATE animals SET species_id = 2 WHERE name LIKE '%mon';
+UPDATE animals SET species_id = 1 WHERE species_id IS NULL;
+
+--Modify your inserted animals so it includes the owner_id value:
+
+UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
+UPDATE animals SET owner_id = 2 WHERE name = 'Gabumon' OR name = 'Pikachu';
+UPDATE animals SET owner_id = 3 WHERE name = 'Devimon' OR name = 'Plantmon';
+UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
+UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon', 'Boarmon');
```
Comment (Munsa1) [inline queries.sql]: - [ ] Kindly ensure that these statements are moved to `data.sql` as they have to do with setting data in the tables

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @yodit93, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Written All Queries :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (yodit93) [reply queries.sql]: Ok, I will correct it.
Thanks

File: queries.sql (hunk 1 above)
Comment (yodit93) [reply queries.sql]: Ok, I will update it. 
Thanks

**Review by christianbiring1 — APPROVED** [review]
Hi @yodit93 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by yodit93** [comment]
Thank you !

**Pull request merged**